components/mixins/opportunity/opportunity-person-components.schema.json

```diff
@@ -0,0 +1,66 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/opportunity-person-components",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Opportunity Person Relation Components",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/classes/opportunity-person"
+  ],
+  "description": "XDM business opportunity person relation component details.",
```

Comment:
This description does not tell me when to use this mixin. Please describe the use-case when this mixin is used.